packages/react/src/components/MapCard/README.md

```diff
@@ -0,0 +1,342 @@
+# `MapCard` component
```

Comment:
I'm not able to see this in the storybook. It might be related to the Experimental sections being duplicated?

Reply:
Ahh, good catch. I need to attach the readme to the using `withReadme`